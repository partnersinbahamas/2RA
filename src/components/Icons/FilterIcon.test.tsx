import React from 'react';
import { render } from '@testing-library/react';
import FilterIcon from './FilterIcon';

describe('FilterIcon', () => {
  it('renders small', () => {
    const { container } = render(<FilterIcon size="SMALL" />);
    expect(container).toMatchSnapshot();
  });
  it('renders medium', () => {
    const { container } = render(<FilterIcon size="MEDIUM" />);
    expect(container).toMatchSnapshot();
  });
  it('renders large', () => {
    const { container } = render(<FilterIcon size="LARGE" />);
    expect(container).toMatchSnapshot();
  });
  it('renders default', () => {
    const { container } = render(<FilterIcon />);
    expect(container).toMatchSnapshot();
  });
});
