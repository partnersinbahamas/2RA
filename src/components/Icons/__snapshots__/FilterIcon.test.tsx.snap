// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterIcon renders default 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="0 0 28 26"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M8.031 17.9689C8.92159 17.9728 9.78561 18.2727 10.4871 18.8214C11.1886 19.3701 11.6877 20.1365 11.906 20.9999H27C27.563 20.9999 28 21.4379 28 21.9999C28 22.5309 27.562 22.9999 27 22.9999H11.875C11.6552 23.8515 11.1581 24.6057 10.462 25.1434C9.76587 25.681 8.91054 25.9715 8.031 25.9689C6.156 25.9689 4.594 24.7189 4.156 22.9999H1C0.437 22.9999 0 22.5309 0 21.9999C0 21.4379 0.438 20.9999 1 20.9999H4.125C4.34598 20.1321 4.85013 19.3628 5.55761 18.8138C6.2651 18.2648 7.13549 17.9675 8.031 17.9689ZM8.031 19.9689C6.906 19.9689 6.031 20.8749 6.031 21.9689C6.031 23.0629 6.906 23.9689 8.031 23.9689C9 23.9689 10.031 23.1879 10.031 21.9689C10.031 20.8749 9.125 19.9689 8.031 19.9689ZM19.937 8.99989C20.8243 8.99859 21.6869 9.29274 22.3885 9.83595C23.0902 10.3792 23.591 11.1405 23.812 11.9999H27C27.563 11.9999 28 12.4379 28 12.9999C28 13.5309 27.562 13.9999 27 13.9999H23.812C23.375 15.7189 21.812 16.9999 19.937 16.9999C19.0497 17.0012 18.1871 16.7071 17.4855 16.1638C16.7838 15.6206 16.283 14.8593 16.062 13.9999H0.938C0.438 13.9999 0.000999928 13.5309 0.000999928 12.9999C0.000999928 12.4379 0.439 11.9999 0.939 11.9999H16.063C16.533 10.2809 18.095 8.99989 19.938 8.99989H19.937ZM19.937 10.9999C18.844 10.9999 17.937 11.9059 17.937 12.9999C17.937 14.1249 18.844 14.9999 19.937 14.9999C21.062 14.9999 21.937 14.1249 21.937 12.9999C21.937 11.9059 21.062 10.9999 19.937 10.9999ZM8 -0.000105782C8.88888 -0.00753379 9.75441 0.284322 10.4573 0.82849C11.1602 1.37266 11.6595 2.13749 11.875 2.99989H27C27.563 2.99989 28 3.43789 28 3.99989C28 4.53089 27.562 4.99989 27 4.99989H11.875C11.375 6.78089 9.844 7.99989 8 7.99989C7.11265 8.0012 6.25012 7.70705 5.54848 7.16384C4.84683 6.62063 4.346 5.85928 4.125 4.99989H1C0.437 4.99989 0 4.53089 0 3.99989C0 3.43789 0.438 2.99989 1 2.99989H4.125C4.563 1.24989 6.125 -0.000105782 8 -0.000105782ZM8 1.99989C6.906 1.99989 6 2.87489 6 3.99989C6 5.09389 6.906 5.99989 8 5.99989C9.094 5.99989 10 5.09389 10 3.99989C10 2.90589 9.125 1.99989 8 1.99989Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`FilterIcon renders large 1`] = `
<div>
  <svg
    class="sc-hLQSwg hewGWh"
    fill="currentcolor"
    viewBox="0 0 28 26"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M8.031 17.9689C8.92159 17.9728 9.78561 18.2727 10.4871 18.8214C11.1886 19.3701 11.6877 20.1365 11.906 20.9999H27C27.563 20.9999 28 21.4379 28 21.9999C28 22.5309 27.562 22.9999 27 22.9999H11.875C11.6552 23.8515 11.1581 24.6057 10.462 25.1434C9.76587 25.681 8.91054 25.9715 8.031 25.9689C6.156 25.9689 4.594 24.7189 4.156 22.9999H1C0.437 22.9999 0 22.5309 0 21.9999C0 21.4379 0.438 20.9999 1 20.9999H4.125C4.34598 20.1321 4.85013 19.3628 5.55761 18.8138C6.2651 18.2648 7.13549 17.9675 8.031 17.9689ZM8.031 19.9689C6.906 19.9689 6.031 20.8749 6.031 21.9689C6.031 23.0629 6.906 23.9689 8.031 23.9689C9 23.9689 10.031 23.1879 10.031 21.9689C10.031 20.8749 9.125 19.9689 8.031 19.9689ZM19.937 8.99989C20.8243 8.99859 21.6869 9.29274 22.3885 9.83595C23.0902 10.3792 23.591 11.1405 23.812 11.9999H27C27.563 11.9999 28 12.4379 28 12.9999C28 13.5309 27.562 13.9999 27 13.9999H23.812C23.375 15.7189 21.812 16.9999 19.937 16.9999C19.0497 17.0012 18.1871 16.7071 17.4855 16.1638C16.7838 15.6206 16.283 14.8593 16.062 13.9999H0.938C0.438 13.9999 0.000999928 13.5309 0.000999928 12.9999C0.000999928 12.4379 0.439 11.9999 0.939 11.9999H16.063C16.533 10.2809 18.095 8.99989 19.938 8.99989H19.937ZM19.937 10.9999C18.844 10.9999 17.937 11.9059 17.937 12.9999C17.937 14.1249 18.844 14.9999 19.937 14.9999C21.062 14.9999 21.937 14.1249 21.937 12.9999C21.937 11.9059 21.062 10.9999 19.937 10.9999ZM8 -0.000105782C8.88888 -0.00753379 9.75441 0.284322 10.4573 0.82849C11.1602 1.37266 11.6595 2.13749 11.875 2.99989H27C27.563 2.99989 28 3.43789 28 3.99989C28 4.53089 27.562 4.99989 27 4.99989H11.875C11.375 6.78089 9.844 7.99989 8 7.99989C7.11265 8.0012 6.25012 7.70705 5.54848 7.16384C4.84683 6.62063 4.346 5.85928 4.125 4.99989H1C0.437 4.99989 0 4.53089 0 3.99989C0 3.43789 0.438 2.99989 1 2.99989H4.125C4.563 1.24989 6.125 -0.000105782 8 -0.000105782ZM8 1.99989C6.906 1.99989 6 2.87489 6 3.99989C6 5.09389 6.906 5.99989 8 5.99989C9.094 5.99989 10 5.09389 10 3.99989C10 2.90589 9.125 1.99989 8 1.99989Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`FilterIcon renders medium 1`] = `
<div>
  <svg
    class="sc-dmyCSP dRcSrL"
    fill="currentcolor"
    viewBox="0 0 20 18"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M6.031 9.969C7.75 9.969 9.406 11.187 9.875 13H18.969C19.531 13 19.969 13.469 19.969 14C19.969 14.562 19.531 15 18.969 15H9.906C9.469 16.75 7.906 17.969 6.031 17.969C4.187 17.969 2.594 16.781 2.156 15H1C0.469 15 0 14.562 0 14C0 13.469 0.469 13 1 13H2.156C2.625 11.25 4.219 9.969 6.031 9.969ZM6 12C4.875 12 4 12.938 4 14C4 15.125 4.906 16 6 16C7.094 16 8 15.125 8 14C8 12.906 7.094 12 6 12ZM14.031 0C14.9162 0.00699413 15.7747 0.304262 16.4747 0.846187C17.1747 1.38811 17.6775 2.14475 17.906 3H19C19.563 3 20 3.469 20 4C20 4.563 19.562 5 19 5H17.906C17.469 6.75 15.906 8 14.031 8C12.219 8 10.594 6.781 10.156 5H1C0.437 5 0 4.562 0 4C0 3.469 0.438 3 1 3H10.156C10.625 1.25 12.219 0 14.031 0ZM14 2C12.906 2 12 2.906 12 4C12 5.094 12.906 6 14 6C15.094 6 16 5.125 16 4C16 2.906 15.094 2 14 2Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;

exports[`FilterIcon renders small 1`] = `
<div>
  <svg
    class="sc-guDLey kBBjNb"
    fill="currentcolor"
    viewBox="0 0 16 12"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      clip-rule="evenodd"
      d="M5 6C5.61986 5.99425 6.22567 6.18457 6.73088 6.54376C7.23609 6.90296 7.61483 7.41265 7.813 8H14.937C15.5 8 15.937 8.438 15.937 9C15.937 9.531 15.5 10 14.937 10H7.813C7.60788 10.5805 7.22646 11.0823 6.72209 11.4354C6.21772 11.7884 5.61562 11.975 5 11.969C4.38438 11.975 3.78228 11.7884 3.27791 11.4354C2.77354 11.0823 2.39212 10.5805 2.187 10H1C0.437 10 0 9.531 0 9C0 8.438 0.438 8 1 8H2.188C2.38611 7.41281 2.76471 6.90324 3.26972 6.54406C3.77472 6.18488 4.38031 5.99446 5 6ZM5 7.969C4.437 7.969 4 8.438 4 8.969C4 9.531 4.438 9.969 5 9.969C5.563 9.969 6 9.531 6 8.969C6 8.406 5.531 7.969 5 7.969ZM10.969 0C11.5873 0.000466767 12.1901 0.193229 12.6939 0.551577C13.1978 0.909925 13.5777 1.41611 13.781 2H15C15.469 2 15.969 2.313 16 3C16 3.563 15.562 4 15 4H13.812C13.375 5.156 12.282 6 10.969 6C10.3506 5.99974 9.74751 5.80707 9.24347 5.44871C8.73944 5.09035 8.35938 4.58405 8.156 4H1C0.531 4 0.031 3.656 0 3C0 2.438 0.438 2 1 2H8.156C8.36316 1.41837 8.74416 0.914519 9.24736 0.556754C9.75056 0.198988 10.3516 0.00461922 10.969 0ZM10.969 2C10.438 2 9.969 2.438 9.969 3C9.969 3.563 10.438 4 10.969 4C11.531 4 11.969 3.563 11.969 3C11.969 2.438 11.5 2 10.969 2Z"
      fill-rule="evenodd"
    />
  </svg>
</div>
`;
